pkg/splunk/enterprise/standalone.go

```diff
@@ -123,6 +123,35 @@ func ApplyStandalone(client splcommon.ControllerClient, cr *enterprisev1.Standal
 		return result, err
 	}
 
+	// If we are using appFramework and are scaling up, we should re-populate the
```

Comment:
More of a comment on the code rasther than this PR itself: I think something similar needs to be done for CRs with app framework and ephemeral storage for etc.

Reply:
ephemeral is an interesting use case. I believe we need to :
1. Error out if the App framework is configured with ephemeral storage
2. Also, need to highlight this in the docs.

Reply:
Yeah probably we can fix the ephemeral part as a separate JIRA. Most probably it will have similar fix but I just want to keep things separate.